theme/widget/theme.scss

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+// For licensing, see LICENSE.md or http://ckeditor.com/license
+
+@import '~@ckeditor/ckeditor5-theme-lark/theme/helpers/_colors.scss';
+@import '~@ckeditor/ckeditor5-theme-lark/theme/helpers/_shadow.scss';
+@import '~@ckeditor/ckeditor5-theme-lark/theme/helpers/_states.scss';
+
+.ck-widget {
+	margin: 10px 0;
+	padding: 0;
+
+	&.ck-widget_selected, &.ck-widget_selected:hover {
+		outline: 2px solid #ace;
```

Comment:
These colors `#ace`, `#ddd` and `yellow` should be variables. Or better, they should come from https://github.com/ckeditor/ckeditor5-theme-lark/blob/master/theme/helpers/_colors.scss#L21-L27. Or even better, the widgets could [extend the standard palette](https://github.com/ckeditor/ckeditor5-theme-lark/blob/master/theme/helpers/_colors.scss#L64-L66). We've never used it before, but it could still work:
```scss
@include ck-color-add( ( 
	'widget-selected': #ace, // 
	'widget-blurred': #ddd 
) );
```

then 

```scss
outline: 2px solid ck-color( 'widget-selected' );
```
BTW: We have [`focus` color](https://github.com/ckeditor/ckeditor5-theme-lark/blob/master/theme/helpers/_colors.scss#L23), which I think makes more sense than `#ace`.